recipes/multirtc/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "multirtc" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/multirtc-{{ version }}.tar.gz
+  sha256: 526487600722b67504fca5339d2d609141c8e08f4dfae5bd1be20bf61a9b201f
+
+build:
+  entry_points:
+    - prep_burst = multirtc.prep_burst:main
+    - multirtc = multirtc.multirtc:main
+  skip: true  # [win or py<39]
```

Comment:
Looks like this one can be noarch, no?

It is OK for noarch package to fails on Windows with a missing dependency. We prefer noarch b/c it is faster to build and the package will be ready when the missing dependency is available on Windows.

Reply:
Ok, sounds good - l'll convert to noarch and expect the windows build to fail.